### PR TITLE
New classify billed order without invoice

### PR DESCRIPTION
**Pull request by atm-etienne**
# NEW [*Add conf to allow or not to classify billed a supplier order without invoice*]
New conf to allow or not to classify billed a supplier order without invoice
Add new visible conf to replace the old hide one, to disallow to classify billed an order from itself, without invoice

### CODE REVIEW

**Comment by eldy** [comment]
Can you update merge last version and repush it. It seems your branch is protected and I can't contribute to fix it myself.

File: htdocs/admin/invoice.php [hunk 1]
```diff
@@ -826,6 +826,14 @@
 print ajax_constantonoff('INVOICE_CHECK_POSTERIOR_DATE');
 print '</td></tr>';
 
+
+// Disallow to classify billed an order without invoice
+print '<tr class="oddeven"><td>'.$langs->trans("InvoiceClassifyBilledSupplierOrderWithoutInvoice"). '&nbsp;' ;
+print $form->textwithpicto('', $langs->trans("InvoiceClassifyBilledSupplierOrderWithoutInvoiceHelp"), 1, 'help') . '</td>';
+print '<td class="left" colspan="2">';
+print ajax_constantonoff('WORKFLOW_DISABLE_CLASSIFY_BILLED_FROM_ORDER');
```
Comment (eldy) [inline htdocs/admin/invoice.php:834]: Constant with name WORKFLOW_* are used when an action into a module generate another into another module (and provided by module Workflow).
Here we want to disable a feature in module order that disable ability to change status billed with no interaction in other module. So option should not be named WORKFLOW_*
So can you rename the option into
ORDER_DISABLE_CLASSIFY_BILLED_FROM_ORDER

File: htdocs/admin/supplier_order.php [hunk 2]
```diff
@@ -547,6 +547,14 @@
 print "</td>\n";
 print "</tr>\n";
 
+
+// Disallow to classify billed a supplier order without invoice
+print '<tr class="oddeven"><td>'.$langs->trans("SupplierOrderClassifyBilledWithoutInvoice"). '&nbsp;' ;
+print $form->textwithpicto('', $langs->trans("SupplierOrderClassifyBilledWithoutInvoiceHelp"), 1, 'help') . '</td>';
+print '<td class="left" colspan="2">';
+print ajax_constantonoff('WORKFLOW_DISABLE_CLASSIFY_BILLED_FROM_SUPPLIER_ORDER');
```
Comment (eldy) [inline htdocs/admin/supplier_order.php:555]: To rename into 
SUPPLIER_ORDER_DISABLE_CLASSIFY_BILLED_FROM_ORDER

**Comment by atm-etienne** [comment]
I did a new commit according to your feedback, but a conflict on `htdocs/admin/commande.php` appears, which I don't have while I do the same thing on my local repository, with the same branches, and I can't see what this is about

**Comment by eldy** [comment]
If you make a pull of the develop branch into your working branch, you should get the conflict. I ca'nt fix it myself, it seems you repo is forbidden to external contrib.

**Comment by atm-etienne** [comment]
I did not have conflict while merging on local, but it's now possible to merge

**Pull request merged**